Extension/.scripts/code.ts

```diff
@@ -0,0 +1,40 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import { spawnSync } from 'child_process';
+import { verbose } from '../src/Utility/Text/streams';
+import { $args, $root, $scenario, brightGreen, checkFile, gray, green, pwd } from './common';
+
+import { resolve } from 'path';
+import { getTestInfo } from '../test/common/selectTests';
+import { install, options } from "./vscode";
+
+export { install, reset } from './vscode';
+
+export async function main() {
+    let ti = (await getTestInfo($scenario));
+    if (!ti) {
+        // try using the $arg0 as a scenario
```

Comment:
Is this supposed to be `$arg0` or `$args[0]`?

Reply:
fixed